subprojects/language-native/src/main/java/org/gradle/language/swift/plugins/SwiftBasePlugin.java

```diff
@@ -76,5 +90,25 @@ public void apply(ProjectInternal project) {
         nativeRuntime.extendsFrom(implementation);
         nativeRuntime.setCanBeConsumed(false);
         nativeRuntime.getAttributes().attribute(Usage.USAGE_ATTRIBUTE, project.getObjects().named(Usage.class, Usage.NATIVE_RUNTIME));
+
+        Configuration testImplementation = project.getConfigurations().create(TEST_IMPLEMENTATION);
+        testImplementation.extendsFrom(implementation);
+        testImplementation.setCanBeConsumed(false);
+        testImplementation.setCanBeResolved(false);
+
+        Configuration testImportPath = project.getConfigurations().create(SWIFT_TEST_IMPORT_PATH);
+        testImportPath.extendsFrom(implementation);
```

Comment:
Should extend `testImplementation`. Can you add some test coverage that shows that this is broken and then fix? Can add in a later PR.